flow/e2e/generic/generic_test.go

```diff
@@ -489,3 +489,64 @@ func (s Generic) Test_Schema_Changes_Cutoff_Bug() {
 	env.Cancel(t.Context())
 	e2e.RequireEnvCanceled(t, env)
 }
+
+func (s Generic) Test_Partitioned_Table_Without_Publish_Via_Partition_Root() {
```

Comment:
should test inherited table with & without setting

Reply:
working on it